helm/dagger/.changie.yaml

```diff
@@ -33,6 +33,34 @@ kinds:
   - label: Security
   - label: Dependencies
 footerFormat: |-
+  {{- $contributorSet := dict }}
+  {{- $maintainers := list "dependabot[bot]" }}
+  {{- range $member := .Env.MAINTAINERS | default "[]" | fromJson }}
+    {{- $member := lower $member }}
+    {{- $maintainers = append $maintainers $member }}
+  {{- end }}
+  {{- range $change := .Changes }}
+    {{- $authorList := splitList " " $change.Custom.Author }}
+    {{- range $author := $authorList }}
+      {{- if not (has $author $maintainers) }}
+        {{- $contributorSet = set $contributorSet $author true }}
+      {{- end }}
+    {{- end }}
+  {{- end }}
+  {{- if $contributorSet -}}
+  ### Contributors
+  Special thanks to our external contributors this release!
+  {{- $contributors := list }}
```

Comment:
nit: keys $myDict | sortAlpha ?

Reply:
Yeah, I updated to take your suggestion :tada: